example/src/main/kotlin/com/expedia/graphql/sample/directives/StringEvalSchemaDirectiveWiring.kt

```diff
@@ -34,17 +28,19 @@ class StringEvalDirectiveWiring : DirectiveWiring {
                     newArguments[graphQLArgumentType.name] = graphQLArgumentType.defaultValue
                 }
             }
-            val newEnv = DataFetchingEnvironmentBuilder.newDataFetchingEnvironment(dataEnv)
+            // NOTE: this relies on internal graphql-java API and may break in the future
```

Comment:
Are there any portions of the library code that also rely in `graphql-java` internals? That, in part, is what broke compatibility with v12

Reply:
`graphql-kotlin` builds on top of `graphql-java` so this comment doesn't really make much sense and I'll remove it.